sos/plugins/collectd.py

```diff
@@ -0,0 +1,45 @@
+# Copyright (C) 2016 Archit Sharma <archit.sh@redhat.com>
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+
+import re
+from sos.plugins import Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin
+
+
+class Collectd(Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin):
+    '''
+    Collectd config collector
+    '''
+    plugin_name = "collectd"
+    profiles = ('services', 'webserver')
+
+    files = (
+        '/etc/collectd.conf',
```

Comment:
Absolutely - there are a few other plugins that do similar things. You can define a variable at class or file scope and use that to avoid repeating yourself with file, package or other lists.